app/models/iteration/analysis.rb

```diff
@@ -20,15 +20,15 @@ def ops_errored?
   end
 
   def approved?
-    status == :approved
+    status == :approve
```

Comment:
The statuses used did not match [the analyzer interface](https://github.com/exercism/automated-analysis/blob/master/docs/analyzers/interface.md#output-format).

Reply:
I think my plan was to change the interface 🙂 

Let's chat about this at some point soon. Certainly `refer_to_mentor` doesn't make sense in v3, but I also wonder about how statuses and comments should work generally.